### PR TITLE
Connect server to values page

### PR DESCRIPTION
**Pull request by SYChJung**
Checkout the [Engineer Workflow Doc](https://docs.google.com/presentation/d/1pdzuHG-yO7_qW6jiByaGTLztKZ6b72lBNqSdNT5WZQ8/edit) to make sure you're contributing correctly! :)

**[Tickets](https://trello.com/b/dYOQKJGv/reclass)** 
Trello Ticket: https://trello.com/c/O12d8SWT/386-connect-server-to-values-page
Additional Notes:

**[Dependencies](https://docs.google.com/document/d/1CM6OA2nlhI3DHnHdDEgySukx_tZ_wsG-C2YgraZRvJA/edit#heading=h.czpk4cbi7z4g)**

**[Explain Impact on DB](https://docs.google.com/document/d/1CM6OA2nlhI3DHnHdDEgySukx_tZ_wsG-C2YgraZRvJA/edit#heading=h.285lgcs4z84o)**

**[Testing](https://docs.google.com/document/d/1CM6OA2nlhI3DHnHdDEgySukx_tZ_wsG-C2YgraZRvJA/edit#heading=h.rs692kr58cf8)**

How did you test this? List of actions & expected behavior with helpful screenshots for reviewer:

If this change is performance related, please document what tests were / should be done to verify improvement:

### CODE REVIEW

File: src/ModuleList/Value/Value.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  border: 2px solid white;
+  color: white;
+  padding: 10px 32px;
+  text-align: center;
+  font-size: 16px;
+  margin: 1.25rem auto;
+  border-radius: 10px;
+
+  float: right;
+`;
```
Comment (brandontiqui) [inline src/ModuleList/Value/Value.tsx:71]: Can you make a styled.ts file and place these styled components there?  Then import them back into the Value component.

File: src/ModuleList/Value/Value.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
```
Comment (brandontiqui) [inline src/ModuleList/Value/Value.tsx:72]: Can you make a styled.ts file and place these styled components there?  Then import them back into the Value component.

**Review by brandontiqui — CHANGES_REQUESTED** [review]
Nice work!  Let me know if you have questions about moving the styled components into a separate file and importing them back into the Value component.

**Review by scottjr632 — APPROVED** [review]
Really good work John!! If you could just make the changes that @brandontiqui requested or reply back to why you think that those styled-components should stay in that one file before merging! Thanks!!

**Review by brandontiqui — APPROVED** [review]

**Pull request merged**